src/main/java/aeneas/views/BuildLevelView.java

```diff
@@ -93,5 +97,8 @@ public void initialize(URL location, ResourceBundle resources) {
     centerBox.setAlignment(Pos.TOP_RIGHT);
     centerBox.getChildren().add(boardView);
 
+    this.lookup("#saveButton").setOnMouseClicked(
```

Comment:
I did that (as in, I tried that exact pair of lines), but `saveButton` never got initialized, so I wasn't sure why (my best guess was because `saveButton` is inside `centerBox`, but I wasn't sure). Perhaps I am missing something.